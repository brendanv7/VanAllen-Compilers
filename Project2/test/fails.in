{/* comments are still ignored */ int@}$
{}()===!=+-$
"hey there Mr.Unterminated String$
!@#%^&*[]:;'<>,.?/$
cApItAl LeTtErS aRe PeOpLe ToO$
pi = 3.14$
how bout a warning /* eh?$