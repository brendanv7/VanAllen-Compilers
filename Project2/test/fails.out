LEXER -- Lexing program 1...
LEXER -- L_BRACE [ { ] found at (1:1)
LEXER -- INT [ int ] found at (1:33)
LEXER -- ERROR: Unrecognized token [ @ ] at (1:38)
LEXER -- R_BRACE [ } ] found at (1:39)
LEXER -- EOP [ $ ] found at (1:40)
LEXER -- Lexing complete for program 1. Lex failed with 1 error(s).

LEXER -- Lexing program 2...
LEXER -- L_BRACE [ { ] found at (2:1)
LEXER -- R_BRACE [ } ] found at (2:2)
LEXER -- L_PAREN [ ( ] found at (2:3)
LEXER -- R_PAREN [ ) ] found at (2:4)
LEXER -- BOOL_OP [ == ] found at (2:5)
LEXER -- ASSIGN_OP [ = ] found at (2:7)
LEXER -- BOOL_OP [ != ] found at (2:8)
LEXER -- INT_OP [ + ] found at (2:10)
LEXER -- ERROR: Unrecognized token [ - ] at (2:11)
LEXER -- EOP [ $ ] found at (2:12)
LEXER -- Lexing complete for program 2. Lex failed with 1 error(s).

LEXER -- Lexing program 3...
LEXER -- QUOTE [ " ] found at (3:1)
LEXER -- CHAR [ h ] found at (3:2)
LEXER -- CHAR [ e ] found at (3:3)
LEXER -- CHAR [ y ] found at (3:4)
LEXER -- SPACE [   ] found at (3:5)
LEXER -- CHAR [ t ] found at (3:6)
LEXER -- CHAR [ h ] found at (3:7)
LEXER -- CHAR [ e ] found at (3:8)
LEXER -- CHAR [ r ] found at (3:9)
LEXER -- CHAR [ e ] found at (3:10)
LEXER -- SPACE [   ] found at (3:11)
LEXER -- ERROR: Illegal symbol in String [ M ] at (3:12)
LEXER -- CHAR [ r ] found at (3:13)
LEXER -- ERROR: Illegal symbol in String [ . ] at (3:14)
LEXER -- ERROR: Illegal symbol in String [ U ] at (3:15)
LEXER -- CHAR [ n ] found at (3:16)
LEXER -- CHAR [ t ] found at (3:17)
LEXER -- CHAR [ e ] found at (3:18)
LEXER -- CHAR [ r ] found at (3:19)
LEXER -- CHAR [ m ] found at (3:20)
LEXER -- CHAR [ i ] found at (3:21)
LEXER -- CHAR [ n ] found at (3:22)
LEXER -- CHAR [ a ] found at (3:23)
LEXER -- CHAR [ t ] found at (3:24)
LEXER -- CHAR [ e ] found at (3:25)
LEXER -- CHAR [ d ] found at (3:26)
LEXER -- SPACE [   ] found at (3:27)
LEXER -- ERROR: Illegal symbol in String [ S ] at (3:28)
LEXER -- CHAR [ t ] found at (3:29)
LEXER -- CHAR [ r ] found at (3:30)
LEXER -- CHAR [ i ] found at (3:31)
LEXER -- CHAR [ n ] found at (3:32)
LEXER -- CHAR [ g ] found at (3:33)
LEXER -- EOP [ $ ] found at (3:34)
LEXER -- ERROR: Unterminated string at (3:34)
LEXER -- Lexing complete for program 3. Lex failed with 5 error(s).

LEXER -- Lexing program 4...
LEXER -- ERROR: Unrecognized token [ ! ] at (4:1)
LEXER -- ERROR: Unrecognized token [ @ ] at (4:2)
LEXER -- ERROR: Unrecognized token [ # ] at (4:3)
LEXER -- ERROR: Unrecognized token [ % ] at (4:4)
LEXER -- ERROR: Unrecognized token [ ^ ] at (4:5)
LEXER -- ERROR: Unrecognized token [ & ] at (4:6)
LEXER -- ERROR: Unrecognized token [ * ] at (4:7)
LEXER -- ERROR: Unrecognized token [ [ ] at (4:8)
LEXER -- ERROR: Unrecognized token [ ] ] at (4:9)
LEXER -- ERROR: Unrecognized token [ : ] at (4:10)
LEXER -- ERROR: Unrecognized token [ ; ] at (4:11)
LEXER -- ERROR: Unrecognized token [ ' ] at (4:12)
LEXER -- ERROR: Unrecognized token [ < ] at (4:13)
LEXER -- ERROR: Unrecognized token [ > ] at (4:14)
LEXER -- ERROR: Unrecognized token [ , ] at (4:15)
LEXER -- ERROR: Unrecognized token [ . ] at (4:16)
LEXER -- ERROR: Unrecognized token [ ? ] at (4:17)
LEXER -- ERROR: Unrecognized token [ / ] at (4:18)
LEXER -- EOP [ $ ] found at (4:19)
LEXER -- Lexing complete for program 4. Lex failed with 18 error(s).

LEXER -- Lexing program 5...
LEXER -- CHAR [ c ] found at (5:1)
LEXER -- ERROR: Unrecognized token [ A ] at (5:2)
LEXER -- CHAR [ p ] found at (5:3)
LEXER -- ERROR: Unrecognized token [ I ] at (5:4)
LEXER -- CHAR [ t ] found at (5:5)
LEXER -- ERROR: Unrecognized token [ A ] at (5:6)
LEXER -- CHAR [ l ] found at (5:7)
LEXER -- ERROR: Unrecognized token [ L ] at (5:9)
LEXER -- CHAR [ e ] found at (5:10)
LEXER -- ERROR: Unrecognized token [ T ] at (5:11)
LEXER -- CHAR [ t ] found at (5:12)
LEXER -- ERROR: Unrecognized token [ E ] at (5:13)
LEXER -- CHAR [ r ] found at (5:14)
LEXER -- ERROR: Unrecognized token [ S ] at (5:15)
LEXER -- CHAR [ a ] found at (5:17)
LEXER -- ERROR: Unrecognized token [ R ] at (5:18)
LEXER -- CHAR [ e ] found at (5:19)
LEXER -- ERROR: Unrecognized token [ P ] at (5:21)
LEXER -- CHAR [ e ] found at (5:22)
LEXER -- ERROR: Unrecognized token [ O ] at (5:23)
LEXER -- CHAR [ p ] found at (5:24)
LEXER -- ERROR: Unrecognized token [ L ] at (5:25)
LEXER -- CHAR [ e ] found at (5:26)
LEXER -- ERROR: Unrecognized token [ T ] at (5:28)
LEXER -- CHAR [ o ] found at (5:29)
LEXER -- ERROR: Unrecognized token [ O ] at (5:30)
LEXER -- EOP [ $ ] found at (5:31)
LEXER -- Lexing complete for program 5. Lex failed with 13 error(s).

LEXER -- Lexing program 6...
LEXER -- CHAR [ p ] found at (6:1)
LEXER -- CHAR [ i ] found at (6:2)
LEXER -- ASSIGN_OP [ = ] found at (6:4)
LEXER -- DIGIT [ 3 ] found at (6:6)
LEXER -- ERROR: Unrecognized token [ . ] at (6:7)
LEXER -- DIGIT [ 1 ] found at (6:8)
LEXER -- DIGIT [ 4 ] found at (6:9)
LEXER -- EOP [ $ ] found at (6:10)
LEXER -- Lexing complete for program 6. Lex failed with 1 error(s).

LEXER -- Lexing program 7...
LEXER -- CHAR [ h ] found at (7:1)
LEXER -- CHAR [ o ] found at (7:2)
LEXER -- CHAR [ w ] found at (7:3)
LEXER -- CHAR [ b ] found at (7:5)
LEXER -- CHAR [ o ] found at (7:6)
LEXER -- CHAR [ u ] found at (7:7)
LEXER -- CHAR [ t ] found at (7:8)
LEXER -- CHAR [ a ] found at (7:10)
LEXER -- CHAR [ w ] found at (7:12)
LEXER -- CHAR [ a ] found at (7:13)
LEXER -- CHAR [ r ] found at (7:14)
LEXER -- CHAR [ n ] found at (7:15)
LEXER -- CHAR [ i ] found at (7:16)
LEXER -- CHAR [ n ] found at (7:17)
LEXER -- CHAR [ g ] found at (7:18)
LEXER -- WARNING: Program ended in open comment. Consider closing it at (7:25)
LEXER -- WARNING: No EOP character found and end-of-file has been reached. at (7:25)
LEXER -- Lexing complete for program 7. No errors found.

